src/test/resources/com/puppycrawl/tools/checkstyle/meta/inputs/checks/customchecks/PropertyWithNoCodeTagCheck.java

```diff
@@ -0,0 +1,32 @@
+package com.puppycrawl.tools.checkstyle.meta.inputs.checks.customchecks;
```

Comment:
please make it
`com.puppycrawl.tools.checkstyle.meta.inputs.checks.custom`

it will make it more logical and let us avoid usage of extra bad word in whitelist.

Reply:
done